packages/nimble-components/src/rich-text-editor-viewer/specs/README.md

```diff
@@ -0,0 +1,335 @@
+# Nimble Rich Text Editor and Nimble Rich Text Viewer
+
+## Overview
+
+-   The `nimble-rich-text-editor` component is a text editing tool that offers a comprehensive range of functionalities for
+    incorporating rich text content. It provides users with the ability to apply different text styles and formats to their content.
+-   The `nimble-rich-text-viewer` component is way to view all the rich text content.
+
+### Background
+
+[Nimble issue #1288](https://github.com/ni/nimble/issues/1288)
+
+[Comments UI mockup](https://www.figma.com/file/Q5SU1OwrnD08keon3zObRX/SystemLink?type=design&node-id=6280-94045)
+
+### Non-goals
+
+-   The visual and interaction design for `nimble-rich-text-editor` and `nimble-rich-text-viewer` is out of scope for this spec component.
```

Comment:
Let's elaborate this more specifically. 
1. ViD and IxD are out of scope for these components in nimble design system  
2. However we have an IxD workflow for SLE application (the figma mockup you linked above) that we will rely on.